contact.css

```diff
@@ -0,0 +1,101 @@
+body{
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: linear-gradient(to top,rgba(0, 0, 0, 0.767),rgba(0, 0, 0, 0.74)),url("images/gh.jpg");
+    width: 100%;
+    height: 600px;
+  }
+  
+  .container{
+    box-shadow: 10px 10px 5px rgba(0, 0, 0, 0.76);
+    background-position: center;
+    width: 400px ;
+    height:270px;
+    margin-left: 400px;
+    margin-top: 100px;
+    padding-bottom: 59px;
+    position: relative;
+  }
+
+  p{
+    text-decoration: none;
+    text-align: center;
+    justify-content: center;
+    font-size: 22px;
+    color: white;
+    font-weight: bolder;
+    text-transform: uppercase;
+  }
+
+  /*.submission-form{
+    width: 200px;
+    margin: 0px;
+    border: 10px rgba(0, 0, 0, 0);
+    padding: 0px;
```

Comment:
- [ ] Please removed all commented code from your code base to make your code clean and easy to maintain